app/templates/events/view/tickets/access.hbs

```diff
@@ -0,0 +1,74 @@
+{{#if device.isMobile}}
+  <br><br>
+{{/if}}
+<div class="ui {{if device.isMobile 'centered' ''}} grid">
+  <div class="{{if device.isMobile '' 'row'}}">
+    <h1 class="ui header {{if device.isMobile 'sixteen' 'six'}} wide column">{{t 'Access Codes'}}</h1>
+    <div class="{{if device.isMobile 'sixteen' 'ten'}} wide column right aligned"><button class="ui button primary">Create an access code</button></div>
+  </div>
+  <div class="{{if device.isMobile '' 'row'}}">
+    {{#if device.isMobile}}
+      <br>
+    {{/if}}
+    <div class="ui white labels {{if device.isTablet 'center aligned three' 'six'}} wide column">
+      <div class="ui label button">{{t 'All'}}</div>
+      <div class="ui label button">{{t 'Active'}}</div>
+      <div class="ui label button">{{t 'Inactive'}}</div>
+    </div>
+    {{#if device.isMobile}}
+      <br>
+    {{/if}}
+    <div class="{{if device.isTablet 'thirteen' 'ten'}} wide column right aligned">
+      <div class="ui search">
+        <div class="ui icon input">
+          <input class="prompt" type="text" placeholder="Search">
+          <i class="search icon"></i>
+        </div>
+        <div class="results"></div>
+        &nbsp;&nbsp;
+        {{#if device.isMobile}}
+          <br><br>
+        {{/if}}
+        <span>{{t 'Show'}}</span>
+        {{#ui-dropdown class='selection'}}
+          <div class="default text"></div>
+          <i class="dropdown icon"></i>
+          <div class="menu">
+            <div class="item" data-value="0">{{t '0'}}</div>
+            <div class="item" data-value="5">{{t '5'}}</div>
+            <div class="item" data-value="20">{{t '20'}}</div>
+            <div class="item" data-value="100">{{t '100'}}</div>
+            <div class="item" data-value="200">{{t '200'}}</div>
+          </div>
+        {{/ui-dropdown}}
+        <span>{{t 'entries'}}</span>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <table class="ui very basic table">
+      <thead>
+        <tr>
+          <th>{{t '#'}}</th>
+          <th>{{t 'Access Code'}}</th>
+          <th>{{t 'Access Code URL'}}</th>
+          <th>{{t 'Accessible Ticket'}}</th>
+          <th>{{t 'Validity'}}</th>
+          <th>{{t 'Status'}}</th>
+          <th>{{t 'Action'}}</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>{{t '1'}}</td>
```

Comment:
Get rid of the translation helper. Do not use it on dummy data. Lines 63-69. Remove tranlsation helpers